### DISCUSSION TITLE
OpenRGB vs Corsair Global Mutex

### DISCUSSION THREAD

**Discussion by Fr0stX76** [EvanMulawski/FanControl.CorsairLink#49] [General]
Hi Evan,

I noticed that you put OpenRGB as not using a global mutex in your readme. I wanted to point out that they have added it as an option a couple months ago. See here:

https://gitlab.com/CalcProgrammer1/OpenRGB/-/merge_requests/1436

**Comment by Fr0stX76** [comment]
Nevermind, they have a mutex on the smbus, which I dont think will avoid conflict on Corsairlink communication...

**Comment by Fr0stX76** [comment]
Well look like it finally happened! I don't own a commander core, so cannot confirm, but it's what they claim...

https://gitlab.com/CalcProgrammer1/OpenRGB/-/commit/6c8da3a04f14490f8d944dfa9471386c697681cc

**Reply by EvanMulawski** [reply]
I've authored a [merge request](https://gitlab.com/CalcProgrammer1/OpenRGB/-/merge_requests/2030) that improves that implementation and adds it to the Lighting Node family (Commander PRO) and the Hydro Platinum family.
